src/containers/Layout/Layout.tsx

```diff
@@ -43,7 +44,9 @@ const Layout = () => {
     shallow,
   );
   const { updateSearchQuery, resetSearchQuery } = useSearchQueryUpdater();
-  const isLoggedIn = !!useAccountStore((state) => state.user);
+  const user = useAccountStore(({ user }) => user, shallow);
```

Comment:
There is no need to shallow-check the `user` property because it should be immutable. 

```suggestion
  const user = useAccountStore(({ user }) => user);
```